Readme.md

```diff
@@ -78,7 +78,7 @@ const fn = match("/foo/:bar");
 
 ## PathToRegexp
 
-The `pathToRegexp` function returns a regular expression for matching strings against paths. It
+The `pathToRegexp` function returns the `regexp` for matching strings against paths, and an array of `keys` for understanding the `RegExp#exec` matches.
```

Comment:
```suggestion
The `pathToRegexp` function returns the `RegExp` for matching strings against paths, and an array of `keys` for understanding the `RegExp#exec` matches.
```

Reply:
It was lower case intentionally to match the key returned.